shell/platform/tizen/external_texture_pixel_gl_stub.cc

```diff
@@ -0,0 +1,77 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/tizen/external_texture_pixel_gl.h"
+
+#ifdef TIZEN_RENDERER_EVAS_GL
+#undef EFL_BETA_API_SUPPORT
+#include "tizen_evas_gl_helper.h"
+extern Evas_GL* g_evas_gl;
+EVAS_GL_GLOBAL_GLES3_DECLARE();
+#else
+#include <EGL/egl.h>
+#include <EGL/eglext.h>
+#include <GLES2/gl2ext.h>
+#include <GLES3/gl32.h>
+#endif
+
+namespace flutter {
+
+ExternalTexturePixelGL::ExternalTexturePixelGL(
```

Comment:
If you add a `_stub.cc` file which has a concrete implementation, then it's not "stub" any more. You might have to split the class into two (e.g. `ExternalTexturePixelGLDevice` and `ExternalTexturePixelGLDesktop`) or consider combining the implementations into a single file if the implementations are not very different (the two files look almost identical as far as I can see).

Reply:
You're right. I'll remove it now. It is better to prepare ``ExternalTexturePixelGLDesktop`` in another PR.

Reply:
:scissors:  done!